### DISCUSSION TITLE
树莓派能用么，aarch64的，python3.9.8,整了一天了，版本都弄了好几个，总是会验证失败，直接就retry了，中间都没停留

### DISCUSSION THREAD

**Discussion by keyfall** [houtianze/bypy#645] [General]
![image](https://github.com/houtianze/bypy/assets/21198605/a2cca0b8-09b7-4cc8-a650-a20eeca4fa9a)

**Comment by houtianze** [comment]
应该可以的，是最新版吗？最新版授权部分已经是直连百度服务器授权。

**Reply by keyfall** [reply]
最新版试过，1.8.2,1.6,1.7都试过，1.7还显示“md5全都不通“,我看了一下官网，就是接入授权那里
![image](https://github.com/houtianze/bypy/assets/21198605/6fca4ce8-2de4-477e-9228-d09562292381)

介绍一下我的操作：树莓派4b，官方系统版本(是否无界面不记得了),aarch64，用pip下载的最新bypy，也没有报utf-8问题
授权时候，出现要输入授权码，这里我不是在树莓派输入地址(我一直使用终端操作的),我把地址复制到笔记本上浏览器进行授权的然后生成的授权码复制过去就验证不了，感觉操作没啥问题


我查资料的时候，使用了一个脚本方法就可以进行登录，脚本主要是通过accesstoken,进行访问网盘的api接口，实现上传，这个可以上传的
[脚本文章](https://blog.csdn.net/zhaojiyuan1024/article/details/125465229?spm=1001.2014.3001.5506)

还有一个事情是我在笔记本上的python的bypy库就可以使用，验证一会就好了

请问你有啥思路么

**Comment by houtianze** [comment]
我刚用最新版试了一下，ssh树莓派上授权和上传没有任何问题，你看一下是不是你哪里的步骤出错了。

**Reply by keyfall** [reply]
我重装了树莓派，试了一下，没有问题
